Code/player controls

AwesomeReid opened the pull request:
- Added sprinting and jumping.

- Added a stamina function which limits how long the player can sprint for and has a cooldown time.

- Settings for these features can be controlled through the player control script.

AwesomeReid committed:
Tuned player controls. Added sprinting and jumping.

AwesomeReid committed:
Merge branch 'master' of https://github.com/ForestStreamStudios/Project-Suzanne into code/player-controls

AwesomeReid committed:
Added sprint cooldown

guenterg reviewed (approved):
Looks good. Changes to the meta files shouldn't cause issues.

bariss0102 committed:
Fix broken camera in SampleScene

PlayerMovement script updated to lock cursor. Changes made to player in SampleScene to fix camera rotation, camera reference in script was incorrect.

bariss0102 committed:
Revert "Fix broken camera in SampleScene"

This reverts commit 884aeeafd3888badf99cb554c60802044f0aced4.

bariss0102 committed:
Recommit-Camera Fix

Merged SampleScene from master.  Previous fix caused merge conflict, so it should be fixed now.(hopefully)

bariss0102 committed:
Revert "Recommit-Camera Fix"

This reverts commit dc051d104e1dcd9206e765bd5f8d52ef9e303405.

bariss0102 commented:
Camera in SampleScene doesn't work, and changing the camera reference in SampleScene causes massive merge conflicts. It should work when moved to master as is, so I'm reverting my commits.
